docs/accordion.md

```diff
@@ -112,26 +126,50 @@ The disabled accordion should only be used when the user is taking a clearly def
 
 The `<Accordion />` component has subcomponents available through dot notation.
 
+### Grouping
+
+Container to holds any number of instances of `<Accordion>` components. **Required**
+to apply standard border-radius and box-shadow styling.
+
+```tsx
+import { Accordion } from 'radiance-ui';
+
+<Accordion.Grouping>
+  <Accordion.Container>
+    <Accordion title={<Accordion.Content>First Title</Accordion.Content>}>
+      <Accordion.Content>First Expansion</Accordion.Content>
+    </Accordion>
+  </Accordion.Container>
+  <Accordion.Container>
+    <Accordion title={<Accordion.Content>Second Title</Accordion.Content>}>
+      <Accordion.Content>Second Expansion</Accordion.Content>
+    </Accordion>
+  </Accordion.Container>
+</Accordion.Grouping>;
```

Comment:
Oh interesting. Since there were some usages of `<Accordion>` without the container I had assumed that it just wraps a single `<Accordion>`. I should have considered more strongly that the docs read `Container to hold instances of <Accordion>` even though all the code examples only had one `<Accordion>` present. Let me refactor.

Reply:
@ZeMunchkin Thank you again for catching this. Updated in https://github.com/curology/radiance-ui/pull/371/commits/03af86c950f0805b6c9939d3f5323253bd55e300 (and story markup formatting fixed in https://github.com/curology/radiance-ui/pull/371/commits/7d95b2b3a04dbea5bf8dcfd1c5fd223cdb5cea06). I've also updated the PR description.